src/webui/api/transfercontroller.cpp

```diff
@@ -111,3 +144,21 @@ void TransferController::speedLimitsModeAction()
 {
     setResult(QString::number(BitTorrent::Session::instance()->isAltGlobalSpeedLimitEnabled()));
 }
+
+void TransferController::banPeersAction()
+{
+    checkParams({"peers"});
+
+    const QStringList peers = params()["peers"].split('|');
+    QList<BitTorrent::PeerAddress> peerList;
+    for (const QString &peer : peers) {
+        const BitTorrent::PeerAddress addr = parsePeer(peer.trimmed());
+        if (addr.ip.isNull())
+            throw APIError(APIErrorType::BadParams);
```

Comment:
Why is error here unlike the action above?

Reply:
I'll remove the error here too